src/problem_bank_helpers/problem_bank_helpers.py

```diff
@@ -427,4 +427,30 @@ def choose_el(x, i, j):
         html += "\n".join(elements)
         html += "\n</tr>"
     html += "\n</table>"
-    return html
\ No newline at end of file
+    return html
+
+
+def mc(data, part_num, choices):
+    """
+    Adds multiple choice to data from dictionary
+
+    Args:
+        choices (dict): the multiple-choice dictionary
+
+    Example:
+        options = {
+            'option1 goes here': ['correct', 'Nice work!'],
+            'option2 goes here': ['Incorrect', 'Incorrect, try again!'],
+            ....
+        }
+
+        mc(data2, 1, options)
+
+    """
+    i = 0
+    for key, value in choices.items():
```

Comment:
Try using `enumerate()` to avoid having a separate counter `i`

```{python}
for i, (k, v) in enumerate(example_dict.items()):
    print(i, k, v)
```